src/layout/List/ListComponent.tsx

```diff
@@ -53,6 +54,10 @@ export const ListComponent = ({
     }
   };
 
+  for (const key in tableHeaders) {
+    tableHeaders[key] = getTextResourceAsString(tableHeaders[key]);
+  }
```

Comment:
Maybe we should consider creating a new object with text resource bindings resolved? That way you don't end up overwriting the object you got as a property.